native/exe/ServiceControlLaunch/ServiceControlLaunch.cpp

```diff
@@ -45,16 +47,17 @@ JavaVM *vm;
 void sysOutPrint(const char* cstr, ...)
 {
 	JNIEnv* env;
-	jint threadState = vm->GetEnv((void**)&env, JNI_VERSION_1_2);
+	jint threadState = vm->GetEnv((void**)&env, JNI_VERSION_1_6); // JNI in JRE 1.6 and greater

```

Comment:
If you're changing this to 1.6, you better provide a good reason as I have no idea what kind of breakage this may introduce by doing something like that.

Reply:
Not using anything in JNI beyond 1.2, so no requirement to change other than to be the latest version that is supported in Java 1.7.
No problem changing it back to 1.2